### DISCUSSION TITLE
How to hide any type of page from sidebar and hamburger navbar on mobile?

### DISCUSSION THREAD

**Discussion by vovavasylyna** [imfing/hextra#421] [Q&A]
Hello,

How to hide any type of page from sidebar and hamburger navbar on mobile? I want it to exist and be publicly accessible via URL but only for those who have the link. In other words, I don't want people to find it just via scrolling through my site. 

Best
Volodymyr

**Comment by imfing** [comment]
Check https://imfing.github.io/hextra/docs/guide/configuration/#main-sidebar
